src/BusinessTable.jsx

```diff
@@ -0,0 +1,151 @@
+import './App.css';
+import { useEffect, useState } from 'react';
+import { useBackend } from './contexts/BackendContext';
+import { Table, Thead, Tbody, Tr, Th, Td } from "@chakra-ui/react";
+
+const BusinessTable = () => {
+  const { backend } = useBackend();
+  const [data, setData] = useState([]);
+
+  const getData = async () => {
+    try {
+      const response = await backend.get('/business');
+      setData(response.data);
+    } catch (error) {
+      console.error('Error fetching data:', error);
+    }
+  };
+
+  useEffect(() => {
+    getData();
+  });
+
+  return (
+    <div>
+      <Table variant="striped" colorScheme="teal">
+        <Thead>
+          <Tr>
+            <Th>Id</Th>
+            <Th>Type</Th>
+            <Th>Name</Th>
+            <Th>Street</Th>
+            <Th>Zipcode</Th>
+            <Th>State</Th>
+            <Th>Qb Vendor Name</Th>
+            <Th>Qb City State Zip</Th>
+            <Th>Primary Phone</Th>
+            <Th>Backup Phone</Th>
+            <Th>Primary Email</Th>
+            <Th>Comments</Th>
+            <Th>Fax phone</Th>
+            <Th>Contact name</Th>
+            <Th>Website</Th>
+            <Th>Business hours</Th>
+            <Th>Find out</Th>
+            <Th>Onboarding status</Th>
+            <Th>Join date</Th>
+            <Th>Input type status</Th>
+            <Th>Vendor type</Th>
```

Comment:
for all of your table headers, I recommend creating a static variable at the top of this page and call it TABLE_HEADERS = ["Id", "Name"... ]

use chatgpt to help you do this, and then within here call 

TABLE_HEADERS.map((table_header) => <Th>{table_header}{/Th}

this will make your code look a lot more cleaner and take up less space.